dask_cuda/utils.py

```diff
@@ -649,3 +653,105 @@ def get_gpu_uuid_from_index(device_index=0):
     pynvml.nvmlInit()
     handle = pynvml.nvmlDeviceGetHandleByIndex(device_index)
     return pynvml.nvmlDeviceGetUUID(handle).decode("utf-8")
+
+
+def get_worker_config(dask_worker):
```

Comment:
Why is this function not private (`_get_worker_config`)? Or, conversely, why are the other functions not public?

Reply:
I'm following a pattern used in dask where the fully async function definition is private and the public function is both sync/async.  For example, we do something similar for [all_to_all](https://github.com/rapidsai/dask-cuda/blob/23460f6f7300202a2d3a25ffedf6884755fe6fbc/dask_cuda/utils.py#L444-L470)